spec/models/contact_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Contact, type: :model do
+  describe 'initialize' do
+    context 'when all attributes are valid' do
+      let!(:user) { User.create(username: 'Mike', biography: 'I am a developer', age: 25) }
+
+      subject { Contact.create(name: 'Rudolph', user_id: 1) }
+
+      it 'creates a new contact' do
+        expect { subject }.to change { Contact.count }.by(1)
+      end
+    end
+  end
+
+  describe 'validates' do
+    context 'when name is not present' do
+      subject { Contact.create(name: nil, user_id: 1) }
```

Comment:
You don't need to try to create to check validations, you can instantiate than call `:valid?` method

```suggestion
      subject { Contact.new(name: nil, user_id: 1) }
```